lang/smaily-for-wp-et.po

```diff
@@ -1,22 +1,21 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: smaily\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-10-05 15:41+0300\n"
-"PO-Revision-Date: 2020-10-05 15:41+0300\n"
-"Last-Translator: Smaily <info@smaily.com>\n"
-"Language-Team: Eesti\n"
+"Project-Id-Version: Smaily for WP\n"
+"POT-Creation-Date: 2021-01-06 12:29+0200\n"
+"PO-Revision-Date: 2021-01-06 12:30+0200\n"
+"Last-Translator: tomabel <tom@smaily.com>\n"
+"Language-Team: Sendsmaily OÜ <info@smaily.com>\n"
```

Comment:
I think it would be better, if the `Last-Translator` and `Language-Team` values are more anonymous.

Please change the values to:
```suggestion
"Last-Translator: Smaily <info@smaily.com>\n"
"Language-Team: Smaily <info@smaily.com>\n"
```

To not change the `Last-Translator` header value when you edit the files with Poedit, you need to change your Name and E-mail address to a empty value under Preferences.

Reply:
Changed with b10ef13